plugins/kubernetes-backend/src/cluster-locator/CatalogClusterLocator.test.ts

```diff
@@ -18,39 +18,68 @@ import '@backstage/backend-common';
 import { CatalogClusterLocator } from './CatalogClusterLocator';
 import { CatalogApi } from '@backstage/catalog-client';
 
+const mockCatalogApi = {
+  getEntityByRef: jest.fn(),
+  getEntities: async () => ({
+    items: [
+      {
+        apiVersion: 'version',
+        kind: 'User',
+        metadata: {
+          annotations: {
+            'kubernetes.io/api-server': 'https://apiserver.com',
+            'kubernetes.io/api-server-certificate-authority': 'caData',
+            'kubernetes.io/auth-provider': 'aws',
+            'kubernetes.io/ca-file': 'cool-ca-file',
+            'kubernetes.io/oidc-token-provider': 'google',
+            'kubernetes.io/skip-metrics-lookup': true,
+            'kubernetes.io/skip-tls-verify': true,
```

Comment:
annotations can only be strings actually, so to make sure to test these things properly you probably want to make them `'true'`

Reply:
Ah thats right ill rework it to use the correct value type thanks.

Reply:
Value type has now been changed to string and method has been changed accordingly. Unit test is passing.